lib/sidekiq/dynamic_queues/attributes.rb

```diff
@@ -82,6 +89,12 @@ def expand_queues(queues)
         while q = queue_names.shift
           q = q.to_s
 
+          if q =~ /^(.+)\/([0-9]+)/
```

Comment:
I'm using it on line 94...
Agree on \A, because it reduces cognitive burden to always use it. 
I need a \Z or $ anyway.